src/amp-story-player/amp-story-player-impl.js

```diff
@@ -905,7 +927,7 @@ export class AmpStoryPlayer {
       this.isCircularWrappingEnabled_ &&
       this.isIndexOutofBounds_(this.currentIdx_ + 1)
     ) {
-      this.go(1);
+      this.go(1, 0, {animate: true});
```

Comment:
This code path is only followed when `circular-wrapping` is enabled, it would still not animate in the regular case